pkgs/by-name/ad/addr-book-combine/package.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromSourcehut,
+  ...
+}:
+buildGoModule {
+  pname = "addr-book-combine";
+  version = "2022-12-15";
+
+  src = fetchFromSourcehut {
+    owner = "~jcc";
+    repo = "addr-book-combine";
+    rev = "11696f4726b981c774ad1d4858f1a935ac71e9ac";
+    hash = "sha256-SENur3p5LxMNnjo/+qiVdrEs+i+rI1PT1wYYdLLqWrg=";
+  };
+
+  vendorHash = null;
+
```

Comment:
add passthru.updateScript

Reply:
I tried adding `nix-update-script`, but it fails with
```
nix_update.errors.VersionError: Please specify the version. We can only get the latest version from codeberg/crates.io/gitea/github/gitlab/pypi/savannah/sourcehut/rubygems/npm projects right now
```
which confuses me, since this _is_ on Sourcehut. Do you know the reason for this maybe?

Reply:
I now added `unstableGitUpdater`, which seems to work